packages/reporters/dev-server/src/Server.js

```diff
@@ -179,8 +181,51 @@ export default class Server extends EventEmitter {
     logger.verbose(`Request: ${req.headers.host}${req.originalUrl || req.url}`);
   }
 
+  /**
+   * Load proxy table from package.json and apply them.
+   */
+  async applyProxyTable(app: any) {
+    // avoid skipping project root
+    const fileInRoot: string = path.join(this.options.projectRoot, '_');
+
+    const pkg = await loadConfig(this.options.inputFS, fileInRoot, [
```

Comment:
I'm not sure if the proxy config file should be loaded from the server. Probably better to make it part of the Parcel config? So load it in core or the CLI.

cc @devongovett

Reply:
@DemoorJasper Still not sure why configuration file should be read from Parcel core packages, but if it is needed, how should I define the new option? Should I hold the exported function directly, create a `connect` app, or define a wrapper class?

Reply:
Ow it's a middleware function. Might be strange to load that in core idk

Reply:
Middlerware for proxies only? Or transfer the all implementation about HTTP request handling into Parcel core?

Reply:
Bundler of v1 has middleware function, so should v2 provide the same function (which handles the proxies as well)?